sdk/eventhub/azure-eventhub/samples/async_samples/send_and_receive_amqp_annotated_message_async.py

```diff
@@ -27,13 +32,13 @@ async def send_data_message(producer):
         delivery_annotations=delivery_annotations,
         application_properties=application_properties
     )
-    batch = await producer.create_batch()
+    batch: EventDataBatch = await producer.create_batch()
```

Comment:
don't think we need to annotate this
```suggestion
    batch = await producer.create_batch()
```